test/Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -437,4 +437,32 @@ spec =
             Meta.type_of (_.is_nothing) . should_equal Function
             Meta.type_of (.is_nothing) . should_equal Function
 
+    Test.group "Meta.Type.find"
+        Test.specify "find boolean and use it" <|
+            fqn = "Standard.Base.Data.Boolean.Boolean"
+            typ = Meta.Type.find fqn
+            typ . is_a Meta.Type . should_be_true
+            typ.name . should_equal "Boolean"
+            Meta.Type.find typ.qualified_name . should_equal typ
+
+        Test.specify "find boolean via associated type and use it" <|
+            fqn = "Standard.Base.Data.Boolean"
+            typ = Meta.Type.find fqn
+            typ . is_a Meta.Type . should_be_true
+            typ.name . should_equal "Boolean"
+            typ.qualified_name . should_equal fqn
+            Meta.Type.find typ.qualified_name . should_equal typ
+
+        Test.specify "unknown type" <|
+            Meta.Type.find "Unknown.Type" . should_fail_with Not_Found
+            Meta.Type.find "UnknownType" . should_fail_with Not_Found
+            Meta.Type.find "" . should_fail_with Not_Found
+
+        Test.specify "My ascribed type" <|
+            meta_type = Meta.meta My_Ascribed_Type
+            fqn =  meta_type . qualified_name
+            fqn.contains "Meta_Spec.My_Ascribed_Type" . should_be_true
```

Comment:
We have a more specialized test method for this - that will yield more readable error messages in case of failure - so it is preferred.

```suggestion
            fqn.should_contain "Meta_Spec.My_Ascribed_Type"
```

`should_be_true` has very unclear error messages, I tend to use it only as last resort - tbh I think at some point we should rethink it and maybe introduce some kind of test assert that will be able to contain some message, because `should_be_true` does not say much in case of failure.